### PR TITLE
fix: use create-pull-request action for update checksums

### PR DESCRIPTION
**Pull request by verbanicm**

### CODE REVIEW

File: .github/workflows/update-checksums.yml [hunk 1]
```diff
@@ -29,11 +31,13 @@ jobs:
     steps:
       - name: 'Checkout'
         id: 'checkout'
-        uses: 'actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8' # ratchet:actions/checkout@v3
```
Comment (sethvargo) [inline .github/workflows/update-checksums.yml:32]: Can we upgrade to v4 while we're here?

File: .github/workflows/update-checksums.yml [hunk 2]
```diff
@@ -44,217 +48,27 @@ jobs:
...
-              core.error(err);
-              core.setFailed(`Failed to create commits for pull request branch: ${err}`);
             }
 
+      # Create a pull request with updated files
       - name: 'Create/Update Pull Request'
-        id: 'create-update-pull-request'
         if: '${{ env.CHANGES }}'
-        uses: 'actions/github-script@98814c53be79b1d30f795b907e553d8679345975' # ratchet:actions/github-script@v6
+        uses: 'abcxyz/pkg/.github/actions/create-pull-request@cbbfe9eb1d44a61f1743f8e25e1b0d0fda7f0677' # ratchet:abcxyz/pkg/.github/actions/create-pull-request@main
```
Comment (sethvargo) [inline .github/workflows/update-checksums.yml:63]: I think we should `ratchet:exclude` the stuff we own.

File: terraform-checksums.json [hunk 3]
```diff
@@ -923,20 +923,6 @@
       "binary_checksum": "bac6bb1fb9383249b40afa46b83a4a78fa9f581347195f0e12bd1967d14a4823",
       "os": "linux",
       "arch": "arm64"
-    },
```
Comment (sethvargo) [inline terraform-checksums.json:926]: Why are these being removed?

Comment (verbanicm) [reply terraform-checksums.json:926]: So I can manually trigger and regenerate them. There has to be changes for the workflow to run and release.

File: .github/workflows/update-checksums.yml (hunk 1 above)
Comment (verbanicm) [reply .github/workflows/update-checksums.yml:32]: done

File: .github/workflows/update-checksums.yml (hunk 2 above)
Comment (verbanicm) [reply .github/workflows/update-checksums.yml]: done

**Review by sethvargo — APPROVED** [review]

**Pull request merged**